Driverless AI/machine-learning-experiment-scoring-and-analysis/machine-learning-experiment-scoring-and-analysis.md

```diff
@@ -118,19 +122,19 @@ The training set contains 375k rows, each row representing a loan, and 27 column
 
  **Note:** the actual data in training and test split vary by user, as the data is split randomly. The Test set contains 125k rows, each row representing a loan, and 27 attribute columns representing attributes of each loan.
 
-7\. Verify that there are three datasets, **freddie_mac_500_test**, **freddie_mac_500_train** and **loan_level_500k.csv**:
+9\. Verify that there are three datasets, **freddie_mac_500_test**, **freddie_mac_500_train** and **loan_level_500k.csv**:
 
 ![loan-level-three-datasets](assets/loan-level-three-datasets.jpg)
```

Comment:
@5675sp thanks for pointing that out. It is nice to have another set of eyes look at all the images in case something is missed. I will add this update in the new commit.